code/game/objects/structures/electricchair.dm

```diff
@@ -48,10 +48,10 @@
 	s.set_up(12, 1, src)
 	s.start()
 	if(buckled_mob)
-		buckled_mob.burn_skin(85)
+		buckled_mob.electrocute_act(85, src, 1)

```

Comment:
I guess that's where the 'safety' argument is for?
But now that I set it to 1, human mobs that are electrocuted by the chair don't get the gray overlay on their own sprites, even though they look charred when you examine them.
Is that related to my changes, or is that just a different bug that I had no fault in?